### DISCUSSION TITLE
Old fission bay concepts

### DISCUSSION THREAD

**Discussion by mega00913** [HelixNebulaStudio/RiseOfTheDead#374] [Show and tell]
The new weather system reminded me of some old FB "rain hazard" concepts created during its development last year. Sharing them here for anyone interested :)

https://github.com/HelixNebulaStudio/RiseOfTheDead/assets/162569959/17b52e2b-ed5b-4a23-ba5d-6266aba10009

https://github.com/HelixNebulaStudio/RiseOfTheDead/assets/162569959/3d37b907-7367-41c9-b6ae-93b6085d8b5f

**Comment by GregTheBestest** [comment]
Did rain work similar to fog, just making it harder to see?

**Reply by mega00913** [reply]
Yep! It was also supposed to make the ground slightly slippery (like ice) and decrease the player's body temperature.
Unfortunately this concept was never suggested as I felt it doesn't work outside FB, as most survival maps were indoors.

**Reply by GregTheBestest** [reply]
Really cool concept though. Would like it more than airstrikes 😂

**Reply by mega00913** [reply]
If you have any suggestions/feedback on how to improve the hazard please do let us know :)

**Reply by GregTheBestest** [reply]
Well, the airstrike hazard grows the radius of explosion and damage from explosion the more you play the survival. On later waves the airstrike becomes nightmare for some players as it is being able to hit through the walls and ceilings, as well having enough damage to mostly kill you or leave with really small amount of HP.

I dont really think that this can be enhanced anyhow, but I would say that the radius of explosion should not be able to get as big as it can right now. On somewhat wave 50-60 the airstrike is already able to hit the player on rooftops map while he is inside the building.
